src/python/WMComponent/WorkflowUpdater/WorkflowUpdaterPoller.py

```diff
@@ -26,6 +31,188 @@
 from WMCore.DAOFactory import DAOFactory
 
 
+def findFiles(path, pat):
+    """
+    Find files within given path and matching given pattern.
+    :param path: starting directory path
+    :param pat: match pattern, e.g. pileup name
+    :return: file name of sandbox tarball
+    """
+    files = []
+    for idir, _, _ in os.walk(path):
+        files.extend(glob.glob(os.path.join(idir, pat)))
+    return files
+
+
+def tarMode(tfile, op):
```

Comment:
I don't think this function was actually required to be implemented, given that we have never changed the compression type. Now that it is implemented, I would suggest moving it under the `Utils/` package as well. Potentially in the FileTools module.